test/Microsoft.Dnx.Testing.CommonUtils/DnxSdkFunctionalTestBase.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Dnx.Runtime;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class DnxSdkFunctionalTestBase
+    {
+        public const string SdkVersionForTestingEnvName = "DNX_SDK_VERSION_FOR_TESTING";
+
+        public static string SdkVersionForTesting
+        {
+            get
+            {
+                var sdkVersionForTesting = Environment.GetEnvironmentVariable(SdkVersionForTestingEnvName);
+                // Warning when DNX_SDK_VERSION_FOR_TESTING is not set?
+                return string.IsNullOrEmpty(sdkVersionForTesting) ? "1.0.0-dev" : sdkVersionForTesting;
+            }
+        }
+
+        public static IEnumerable<object[]> DnxSdks
+        {
+            get
+            {
+                return ClrDnxSdks.Concat(CoreClrDnxSdks);
+            }
+        }
+
+        public static IEnumerable<object[]> ClrDnxSdks
+        {
+            get
+            {
+                if (RuntimeEnvironmentHelper.IsWindows)
```

Comment:
I'm a little confused. Is the expected behavior 
1 use mono runtime for mono environment
2 use clr x86/x64 for all other environments
?